lib/developer.js

```diff
@@ -78,6 +78,7 @@ function developer (opts) {
     }, opts);
 
     initialRequest(opts)
+      .then(getListWithPermissions(opts))
```

Comment:
I tried it firstly, but as you're controlling the `num` param in the `checkFinished` method, it'd request the permissions to all  the 49 ( or 48 ) apps, even if the number requested is lower (since the control is done by the `savedApps` array). I tried to change this behavior (adding the `splice` as it's being done in the `getDetailList` didn't worked) doing small changes in methods involved on this, but i couldn't.